charts/seed-bootstrap/dashboards/controllers.json

```
@@ -0,0 +1,1725 @@
+            "instant": true,
+            "interval": "",
+            "legendFormat": "{{controller}}",
+            "refId": "A"
+          }
+        ],
+        "title": "Reconciliation Time Distribution",
+        "type": "piechart"
+      },
+      {
+        "collapsed": false,
+        "datasource": null,
+        "gridPos": {
+          "h": 1,
+          "w": 24,
+          "x": 0,
+          "y": 9
+        },
+        "id": 18,
+        "panels": [],
+        "title": "Reconciliations",
+        "type": "row"
+      },
+      {
+        "datasource": "${datasource}",
+        "description": "Shows the total number of reconciliations per controller.",
+        "fieldConfig": {
+          "defaults": {
+            "color": {
+              "mode": "palette-classic"
+            },
+            "custom": {
+              "axisLabel": "reconciliations/s",
+              "axisPlacement": "auto",
+              "barAlignment": 0,
+              "drawStyle": "line",
+              "fillOpacity": 30,
+              "gradientMode": "opacity",
+              "hideFrom": {
+                "graph": false,
+                "legend": false,
+                "tooltip": false
+              },
+              "lineInterpolation": "smooth",
+              "lineWidth": 1,
+              "pointSize": 5,
+              "scaleDistribution": {
+                "type": "linear"
+              },
+              "showPoints": "auto",
+              "spanNulls": false
+            },
+            "decimals": 4,
+            "mappings": [],
+            "thresholds": {
+              "mode": "absolute",
+              "steps": [
+                {
+                  "color": "green",
+                  "value": null
+                },
+                {
+                  "color": "red",
+                  "value": 80
+                }
+              ]
+            },
+            "unit": "short"
+          },
+          "overrides": []
+        },
+        "gridPos": {
+          "h": 10,
+          "w": 24,
+          "x": 0,
+          "y": 10
+        },
+        "id": 2,
+        "options": {
+          "graph": {},
+          "legend": {
+            "calcs": [
+              "max",
+              "mean",
+              "last"
+            ],
+            "displayMode": "table",
+            "placement": "right"
+          },
+          "tooltipOptions": {
+            "mode": "single"
+          }
+        },
+        "pluginVersion": "7.5.17",
+        "targets": [
+          {
+            "exemplar": true,
+            "expr": "sum(rate(controller_runtime_reconcile_total{job=\"$job\",controller=~\"$controller\"}[$__rate_interval])) by (controller)",
+            "interval": "",
+            "legendFormat": "{{controller}}",
+            "refId": "A"
+          }
+        ],
+        "timeFrom": null,
+        "timeShift": null,
+        "title": "Number of Reconciliations by controller",
```

This is not a counter (as "Number of ..." would indicate). This panel shows a rate.

To make this easier to understand and consistent with the other panel titles you could rename this one:
```suggestion
        "title": "Reconciliation Rate by controller",
```